includes/dbFacile.php

```diff
@@ -39,23 +39,14 @@ function dbQuery($sql, $parameters = array())
         }
     }
 
-    /*
-        if($this->logFile)
-        $time_start = microtime(true);
-     */
-
     $result = mysqli_query($database_link, $fullSql);
-    // sets $this->result
-    /*
-        if($this->logFile) {
-        $time_end = microtime(true);
-        fwrite($this->logFile, date('Y-m-d H:i:s') . "\n" . $fullSql . "\n" . number_format($time_end - $time_start, 8) . " seconds\n\n");
+    if (!$result) {
```

Comment:
So what we do here is check if the mysql log level is INFO or ERROR AND don't contain duplicate entry in the error (we have these and they need to be fixed but will come back to that) and print out any mysql errors OR if the log level is DEBUG then just print out all failed queries.

I've then used this to find all broken queries.